lib-intro.tex

```diff
@@ -72,3 +72,124 @@
 If the semantic requirements of a declaration are not satisfied at the
 point of use, the program is ill-formed, no diagnostic required.
 }
+
+\setcounter{section}{5}
+\setcounter{subsection}{2}
+\setcounter{subsubsection}{1}
+\setcounter{paragraph}{4}
+
+{\color{newclr}
+\rSec4[customization.point.object]{Customization Point Objects}
+\pnum
+A \techterm{customization point object} is an object with a
+literal class type which interacts with user-defined types while
+enforcing semantic requirements on that interaction.
+
+\pnum
+If \tcode{T} is the type of a customization point object, \tcode{T}
+shall satisfy \tcode{Semiregular<T>()} and
+\tcode{Function<const T, Args..>()} when the types of \tcode{Args..}
```

Comment:
If I'm understanding, your suggestion here is that each customization point object type should specify its results to satisfy a specified concept - as all those currently specified do if we call `Same<void>` the result concept of `swap` - and that this should be a required feature of \techterm{customization point object}? (If I'm misunderstanding and that's *not* your suggestion, I may put it in anyway because it's a great idea.)